src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,52 @@
+package seedu.addressbook.ui;
+
+import java.util.List;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+/**
+ * Formatter will have public methods used by TextUI to format text.
+ * @author unusep
+ *
+ */
+public class Formatter {
+
+    /** Format of indexed list item */
+    private static final String MESSAGE_INDEXED_LIST_ITEM = "\t%1$d. %2$s";
+
+    /** Offset required to convert between 1-indexing and 0-indexing.  */
+    public static final int DISPLAYED_INDEX_OFFSET = 1;
+    
+	/**
+	 * formatPersons takes in a list of persons and a list to store the formattedPersons,
```

Comment:
The start of the header comment does not follow the convention (i.e. first sentence should be something like `Selects next line …` instead of `Select next line` or `This method is for selecting …`)